apps/packages/ui/src/components/Option/KnowledgeQA/KnowledgeQAProvider.tsx

```
@@ -108,10 +123,21 @@ function reducer(state: KnowledgeQAState, action: Action): KnowledgeQAState {
     case "ADD_THREAD":
       return { ...state, threads: [action.payload, ...state.threads] }
     case "SET_PRESET":
+      const presetSettings =
+        action.payload === "custom"
+          ? state.settings
+          : applyRagPreset(action.payload as Exclude<RagPresetName, "custom">)
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)